spec.bs

```diff
@@ -1074,7 +1099,7 @@ A [=part=] has an associated <dfn for=part>suffix</dfn>, a string, initially the
 
 <h4 id=options-header>Options</h4>
 
-An <dfn>options</dfn> [=struct=] contains different settings that control how [=/pattern string=] behaves.  These options originally come from [path-to-regexp](https://github.com/pillarjs/path-to-regexp).  We only include the options that are modified within the URLPattern specification and exclude the other options.  For the purposes of comparison, this specification acts like [path-to-regexp](https://github.com/pillarjs/path-to-regexp) where `sensitive`, `strict`, `start`, and `end` are always set to true.
+An <dfn>options</dfn> [=struct=] contains different settings that control how [=/pattern string=] behaves.  These options originally come from [path-to-regexp](https://github.com/pillarjs/path-to-regexp).  We only include the options that are modified within the URLPattern specification and exclude the other options.  For the purposes of comparison, this specification acts like [path-to-regexp](https://github.com/pillarjs/path-to-regexp) where `ignoreCase` (the original [path-to-regexp](https://github.com/pillarjs/path-to-regexp) options is `sensitive` that is synonymous to negated `ignoreCase`), `strict`, `start`, and `end`.  All the aforementioned properties are always set to true except `ignoreCase` which is set to false.
```

Comment:
You need another paragraph below that includes a `<dfn for=options>ignore case</dfn>` to define the attribute.  Then you can reference it like `[=options/ignore case=]`.  And I think using the separate words instead of camel case to match style would be slightly preferred.

The second paragraph also explicitly needs to say `[=options/ignore case=] default to false` or we need to update the various definitions of `[=default options=]`, etc, to set the value to false.